marklogic-data-hub-central/ui/src/components/modeling/property-table/property-table.module.scss

```diff
@@ -0,0 +1,29 @@
+.icon {
+  font-size: 21px;
+  margin-left: 11px;
+}
+
+.identifierIcon {
+  font-size: 21px;
+  margin-left: 16px;
+}
+
+.addIcon {
+  cursor: pointer;
+  font-size: 21px;
+  margin-left: 6px;
+  color: #5B69AF;
+}
+
+.addBtnContainer {
```

Comment:
Unless we don't control the term "addBtn", I highly, highly, highly recommend avoiding abbreviations like this. They don't serve any purpose other than to make code a little harder to read. Plus, more verbose names are easier to search for when looking for references.